src/App.js

```diff
@@ -0,0 +1,90 @@
+// src/App.js
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import Home from './components/home';
+import Signup from './auth/signup';
+import Login from './auth/login';
+import { checkLoginStatus } from './redux/loginSlice';
+import Navbar from './components/navbar';
+import AddItem from './components/addItem';
+import ItemDetails from './components/itemDetails';
+import AddReservations from './components/addReservations';
+import DisplayReservations from './components/displayReservations';
+import DeleteItems from './components/deleteItems';
+import './stylesheets/App.css';
+
+function App() {
```

Comment:
- [ ] I was able to reserve in the year 1374:

![image](https://github.com/geekelo/fenaj-schedula/assets/57421823/abe0c5e2-80c5-4f4c-9749-8cba9b61455e)

Kindly, add some validation to make sure your users can only reserve in the future. 👍